services/apps/alcs/src/alcs/admin/application-decision-condition-types/application-decision-condition-types.controller.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { createMock, DeepMocked } from '@golevelup/nestjs-testing';
+import { ConfigModule } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ClsService } from 'nestjs-cls';
+import { mockKeyCloakProviders } from '../../../../test/mocks/mockTypes';
+import { ApplicationDecisionConditionType } from '../../application-decision/application-decision-condition/application-decision-condition-code.entity';
+import { ApplicationDecisionConditionTypesController } from './application-decision-condition-types.controller';
+import { ApplicationDecisionConditionTypesService } from './application-decision-condition-types.service';
+
+describe('ApplicationDecisionConditionTypesController', () => {
+  let controller: ApplicationDecisionConditionTypesController;
+  let mockDecTypesService: DeepMocked<ApplicationDecisionConditionTypesService>;
+
+  beforeEach(async () => {
+    mockDecTypesService = createMock();
+
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [ApplicationDecisionConditionTypesController],
+      providers: [
+        {
+          provide: ApplicationDecisionConditionTypesService,
+          useValue: mockDecTypesService,
+        },
+        {
+          provide: ClsService,
+          useValue: {},
+        },
+        ...mockKeyCloakProviders,
+      ],
+      imports: [ConfigModule],
+    }).compile();
+
+    controller = module.get<ApplicationDecisionConditionTypesController>(
+      ApplicationDecisionConditionTypesController,
+    );
+  });
+
+  it('should be defined', () => {
+    expect(controller).toBeDefined();
+  });
+
+  it('should call out to service when fetching ceo criterion', async () => {
```

Comment:
looks like a typo, should be 'condition type'. This is in all tests for this file

Reply:
Fixed